libraries/objectDefaultFiles/gl-worker.js

```diff
@@ -51,24 +54,43 @@ function makeStub(functionName) {
       }
     }
 
+    if (functionName === 'texImage2D') {
```

Comment:
just curious, do you anticipate many cases that need to be handled individually like this, or is texImage2D an outlier?

Reply:
It's the image part of texImage2D that causes an issue so I think it's an outlier. There's plenty of room for things to require special-casing in other ways (and I had to modify a tool's three.min.js file to work around one)